superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -123,32 +123,11 @@ const LabelContainer = (props: {
   className: string;
 }) => {
   const labelRef = useRef<HTMLDivElement>(null);
-  const [showTooltip, setShowTooltip] = useState(true);
-  const isLabelTruncated = () =>
-    !!(
-      labelRef &&
-      labelRef.current &&
-      labelRef.current.scrollWidth > labelRef.current.clientWidth
-    );
```

Comment:
Hey @zhaoyongjie,
It appears that with removing this part we lose the ability to show the tooltip when the text is truncated. This ability was requested several times in the past in order to show the full label even when it does not fit the panel width. I am not sure if there is a consensus on whether it should now be removed. Please watch the video below that shows that no tooltip is sho

https://user-images.githubusercontent.com/60598000/149977435-2e4e9e15-b02a-44a5-a1d4-dae409893b41.mp4

wn when the text in the panel is truncated. Thank you!

Reply:
Nice catch! I will update it.

Reply:
@geido I have fixed this. Please reveiw again, Thanks!

![image](https://user-images.githubusercontent.com/2016594/150163419-f259ed36-b60e-4074-a203-e925dcacb54a.png)